css/index.css

```diff
@@ -0,0 +1,303 @@
+/* Background */
+* {
+  box-sizing: border-box;
+}
+/* .underwater {
+  width: 100;
+  height: 100vh;
+  background-image: url(images\background.png);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  overflow: hidden;
+} */
+
+/* song */
+.iframe {
+  position: absolute;
+  top: 0px;
+  right: 500px;
+  left: 0px;
+  /* width: 500px;
+  height: 0px; */
+}
+
+.welcome {
+  color: whitesmoke;
+  padding-top: 300px;
+  padding-bottom: 500px;
+  font-size: larger;
+  font-weight: 500;
+  text-align: center;
+  font-family: "Lobster";
+}
+
+.direction-buttons {
+  color: whitesmoke;
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  font-family: "Lobster";
+}
+.direction-buttons ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.direction-buttons li {
+  display: inline;
+  margin-right: 20px;
+}
+.direction-buttons a {
+  text-decoration: none;
+  color: whitesmoke;
+}
+
+.direction-buttons a:hover {
+  color: whitesmoke;
+}
+
+.body {
+  background-color: #006994;
+  color: rgb(0, 0, 0);
+  font-family: "Courier New", Courier, monospace;
+}
+
+h2 {
+  font-family: Georgia, "Times New Roman", Times, serif;
+  color: rgb(57, 150, 119);
+  font-weight: bold;
```

Comment:
I think it's good to experiment with all the different methods, but in the long run you'd ideally pick one type of color definition and stick to that as much as possible